src/modules/cpu/host_advanced_augmentations.hpp

```diff
@@ -2620,4 +2620,144 @@ RppStatus glitch_host_batch(T* srcPtr, RppiSize *batch_srcSize, RppiSize *batch_
     return RPP_SUCCESS;
 }
 
+
+/**************** ricap - random image crop and patch ***************/
+
+template <typename T>
+RppStatus ricap_host_batch(T* srcPtr, RppiSize *batch_srcSize, RppiSize *batch_srcSizeMax, T* dstPtr,
+                                    Rpp32u *permutedIndices1, Rpp32u *permutedIndices2, Rpp32u *permutedIndices3, Rpp32u *permutedIndices4,
+                                    Rpp32u *cropRegion1, Rpp32u *cropRegion2, Rpp32u *cropRegion3, Rpp32u *cropRegion4,
+                                    Rpp32u outputFormatToggle, Rpp32u nbatchSize,
+                                    RppiChnFormat chnFormat, Rpp32u channel)
+{
+
+    if(chnFormat == RPPI_CHN_PLANAR)
+    {
+        //to planar
+        Rpp32u cropBufferinBytes[4];
```

Comment:
Follow consistency in camelCase - Capitalize the I in 'in' - cropBufferInBytes for all occurrences in the file